docker/docker-compose.yml

```diff
@@ -1,11 +1,11 @@
 x-app: &app
-  image: cas:${TARGET:-dev-build}-1.1
+  image: cas:${TARGET:-dev-build}-2.0
   build:
     context: .
     target: ${TARGET:-dev-build}
     dockerfile: docker/app/Dockerfile
     args:
-      BUILD_TAG: 3.3.7-alpine3.20
+      BUILD_TAG: 3.3.8-slim-bookworm
       PG_MAJOR: '12'
```

Comment:
Looked quickly, and these PG args I think aren't used. I'd remove any that aren't in use.